t/math/declare.xml

```diff
@@ -924,16 +924,16 @@
     </tags>
     <title><tag close=" ">6</tag>Notations?</title>
     <declare xml:id="S6.XMD1">
-      <tag>Baz</tag>
+      <text>Baz</text>
     </declare>
     <declare xml:id="S6.XMD2">
-      <tag>Plus</tag>
+      <text>Plus</text>
     </declare>
     <declare xml:id="S6.XMD3">
-      <tag>Foo</tag>
+      <text>Foo</text>
     </declare>
     <declare xml:id="S6.XMD4">
-      <tag>Oddball</tag>
+      <text>Oddball</text>
```

Comment:
Cool! Is there also a test for the new `<tags>` capability? It may double as a nice example of advanced rewrite output that we keep around to avoid regressions.

Reply:
No, actually you'll just get errors (if you used old xml; which is kinda not likely to happen, or?)

Reply:
I was referring to the new capability at:
https://github.com/brucemiller/LaTeXML/pull/1317/files#diff-f56ed2130145887d3624288c57713a2dR156

where `ltx:declare` is allowed a new `definiens` attribute and a `ltx:tags` child. Could be nice to have a test for that?

Reply:
Ah, yesss, well... I'm not sure I want to commit (to coin a phrase) yet to how that attribute gets there...

Reply:
But I forgot to copy my dev version of declare.tex over, which at least did exercise tag more!